### PR TITLE
[16.0][ADD] sale_partner_address_restrict

### PR DESCRIPTION
**Pull request by MarinaAForgeFlow**

### CODE REVIEW

**Review by AaronHForgeFlow — COMMENTED** [review]
Functional tests ok.

I tried to cheat the system by removing the main partner then setting a invoice address and then tried to use a partner that is not a parent. I could do that but when that happens the invoice and delivery address are automatically updated, that is ok.

**Comment by MiquelRForgeFlow** [comment]
Put sale_partner_address_restrict in the commit too.

File: sale_partner_address_restrict/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "summary": "Restrict addresses domain in the sales order form"
+    " taking into account the partner selected",
+    "version": "16.0.1.0.0",
+    "category": "Sales",
+    "website": "https://github.com/OCA/partner-contact",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "installable": True,
+    "depends": [
+        "sale_management",
```
Comment (MiquelRForgeFlow) [inline sale_partner_address_restrict/__manifest__.py]:
```suggestion
        "sale",
```

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM!

**Pull request merged**